core/src/main/kotlin/xtdb/api/IndexerConfig.kt

```diff
@@ -13,9 +14,17 @@ data class IndexerConfig(
     var pageLimit: Long = 1024L,
     var rowsPerBlock: Long = 102400L,
     var flushDuration: Duration = Duration.ofHours(4),
+    var skipTxs: List<LogOffset> = System.getenv("XTDB_SKIP_TXS")?.let(::parseSkipTxsEnv).orEmpty()
```

Comment:
Is there a good place to test this? @jarohen

Reply:
I probably wouldn't worry tbh, but if you want you can make the private fun `internal` and run a few cases through it